tui/satellite/host.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
 
-HIVEMIND_HOST=$(whiptail --inputbox --cancel-button "$CANCEL_BUTTON" --ok-button "$OK_BUTTON" --title "$TITLE_HOST" "$CONTENT_HOST" 25 80 3>&1 1>&2 2>&3)
-export HIVEMIND_HOST
+HIVEMIND_HOST=$(whiptail --inputbox --cancel-button "$BACK_BUTTON" --ok-button "$OK_BUTTON" --title "$TITLE_HOST" "$CONTENT_HOST" 25 80 3>&1 1>&2 2>&3)
 
 exit_status=$?
-if [ "$exit_status" -eq 1 ]; then
-  exit 0
-fi
+
+if [ $exit_status == 0 ]; then
```

Comment:
The `"` are missing around `$exist_status` and `-eq` should be used instead of `==`.